comps/retrievers/langchain/vdms/docker/docker_compose_retriever.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+version: "3.8"
+
+services:
+  tei_xeon_service:
+    image: ghcr.io/huggingface/text-embeddings-inference:cpu-1.2
```

Comment:
Please use the latest image ghcr.io/huggingface/text-embeddings-inference:cpu-1.5

Reply:
Done. changed all references for text-embeddings-inference:cpu-1.2 to text-embeddings-inference:cpu-1.5. in the commit that will be pushed shortly.